### DISCUSSION TITLE
About Nx Cloud and gitlab ci example

### DISCUSSION THREAD

**Discussion by hayawata3626** [nrwl/nx#10805] [General]
Nx Cloud is so amazing tools.

I usually use gitlab ci and am trying to configure nx cloud, but there are no examples of nx cloud and gitlab ci in the documentation. 

https://nx.dev/nx-cloud/set-up/set-up-dte#cicd-examples

If you have examples I would like to refer to them.
Are there any plans to add these documents?

**Comment by gmeligio** [comment]
There is a guide here about integrating Nx with Gitlab and talks about Nx Cloud too.

https://nx.dev/recipe/monorepo-ci-gitlab

**Reply by hayawata3626** [reply]
@gmeligio 
I Confirmed! 
You had more documents. Thank you!

**Comment by timothylombrana** [comment]
For posterity [https://nx.dev/recipes/ci/monorepo-ci-gitlab](https://nx.dev/recipes/ci/monorepo-ci-gitlab), I believe.
